playbooks/cloud-provider-openstack-test-occm/run-1.19.yaml

```diff
@@ -48,7 +48,7 @@
         name: install-k3s-vm
       vars:
         devstack_workdir: "{{ global_env.devstack_workdir }}"
-        cpo_branch: "{{ zuul.branch }}"
+        k8s_branch: "{{ k8s_release | default(zuul.branch) }}"
```

Comment:
@lingxiankong etcd versions /go lang versions supported change for each k8s release. Does this prerequisites taken care by k3s?

Reply:
What's the requirement for etcd and golang version? k3s doesn't use etcd. Why do we care about Golang? We have openstack-cloud-controller-manager built with minimum version of Go 1.15, and openstack-cloud-controller-manager talks with k8s API.

Reply:
Ah, right. We are not building k8s unlike before, so shouldnt be required.